src/main/kotlin/snyk/common/lsp/Types.kt

```diff
@@ -201,14 +386,82 @@ data class IssueData(
     @SerializedName("priorityScore") val priorityScore: Int,
     @SerializedName("hasAIFix") val hasAIFix: Boolean,
     @SerializedName("dataFlow") val dataFlow: List<DataFlow>,
+
+    // OSS
```

Comment:
I don't know if there's a better way to do this. I tried making the `additionalData` an `Any` but that breaks the gson serializer. I don't know if we can define our own adapter for this ... https://stackoverflow.com/questions/28319473/gson-same-field-name-different-types?

At the moment I'm using a field from each product type to tell us which product it is and so which format